serializability-checker-compiler-plugin/src/test/scala/org/virtuslab/ash/SerializabilityCheckerCompilerPluginComponentSpec.scala

```diff
@@ -26,6 +26,7 @@ class SerializabilityCheckerCompilerPluginComponentSpec extends AnyFlatSpecLike
   it should "detect lack of serializer trait with Behavior" in {
     val out = SerializabilityCheckerCompiler.compileCode(List(serNoCode, singleBehavior))
     out should include("error")
+    out should include(ClassType.Message.name.toLowerCase)
```

Comment:
Whoops do we have any test cases for errors about event/state not being serializable? 😅